packages/ibm-products-styles/src/components/NotificationsPanel/_notifications-panel.scss

```
@@ -42,32 +43,33 @@ $block-class: #{c4p-settings.$pkg-prefix}--notifications-panel;
   100% {
     opacity: 0;
     // stylelint-disable-next-line carbon/layout-use
-    transform: translateY(-38.5rem); // the height of the notification panel
+    transform: translateY(-$block-size); // the height of the notification panel
   }
 }
 
 .#{$block-class}__container {
-  @include theme($g100);
-
   position: fixed;
   z-index: 2;
   overflow: auto;
-  background-color: $background;
+  background-color: $layer-01;
+  border-block-end: 1px solid $border-subtle-02;
+  border-inline-start: 1px solid $border-subtle-02;
```

```suggestion
  border-block-end: 1px solid $border-subtle;
  border-inline-start: 1px solid $border-subtle;
```

i was looking at the border of the switcher, and it looks like its using just `$border-subtle` token
i think we need to follow the same?

---

Hi @devadula-nandan,

Good question. There's no design document to go with this issue, so neither is wrong.

Since the UI Shell's side panel and the NotificationPanel don't have the same styling or entrance/exit animations, I re-used the tokens from Product's [Side Panel](https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-side-panel-sidepanel--slide-in&globals=viewport:basic). It's design and functionality seems closer to the NotificationPanel and includes a drop shadow.